src/bootstrap/src/core/builder/tests.rs

```diff
@@ -1106,16 +1108,16 @@ mod snapshot {
     }
 
     #[test]
+    // Avoids a fail due to not having this line in the snapshot of non-amd64 linux:
+    // [doc] rustc 2 <host> -> std 2 <x86_64-unknown-ferrocene.certified> crates=[core]
+    #[cfg(target_os = "linux")]
```

Comment:
oh, i missed that these tests still run on some platforms. in that case i'm happy to merge. but this cfg is wrong: the thing you care about is a x86-64 host, not whether it's linux or not. 
```suggestion
    #[cfg(target_arch = "x86_64")]
```

Reply:
oh, indeed... didn't think that far